core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -653,6 +667,12 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
     }
   }
 
+  override def updateLogLevel(logLevel: String): Unit = {
```

Comment:
Nit: 
```suggestion
  override def updateExecutorsLogLevel(logLevel: String): Unit = {
```
Also update this please in the base class and  in the SparkContext where it is used.